src/codegen/cppbind.ts

```diff
@@ -0,0 +1,640 @@
+import { SyntaxNode } from "@lezer/common";
```

Comment:
is there an assertion that when the return type is JSValue that check_slow isnt allowed?

Reply:
I thought that would be wrong for JSC__JSValue__getIfPropertyExistsImpl but it looks like that returns propertydeleted rather than zero. so I'll add it.